app/src/main/java/org/merakilearn/ui/home/HomeViewModel.kt

```diff
@@ -14,6 +14,10 @@ class HomeViewModel(private val applicationRepo:ApplicationRepo) : ViewModel() {
         emit(applicationRepo.fetchOtherCourseData())
     }
 
+    fun fetchUpcomingClass(langCode: String?) = liveData {
```

Comment:
Why is this needed here?

Reply:
For langugage filter

Reply:
Yes but this should be handled by DiscoverViewModel, not HomeViewModel @aashish-aadarsh